src/main/java/io/jenkins/plugins/propelo/commons/service/LevelOpsPluginConfigService.java

```diff
@@ -34,10 +39,26 @@ public LevelOpsPluginConfigService() {
                 .build(new CacheLoader<String, LevelOpsConfig>() {
                     @Override
                     public LevelOpsConfig load(String s) throws Exception {
-                        if(!OVERRIDE_CONFIG_FILE.exists()) {
-                            return DEFAULT_LEVELOPS_CONFIG;
-                        }
                         try {
+                            ApplicationType applicationType = PropeloPluginImpl.getInstance().getApplicationType();
+
+                            if (!OVERRIDE_CONFIG_FILE.exists()){
+                                switch (applicationType){
+                                    case SEI_LEGACY:
+                                        return DEFAULT_LEVELOPS_CONFIG;
+                                    case SEI_LEGACY_EU:
+                                        return EU_LEVELOPS_CONFIG;
+                                    case SEI_LEGACY_ASIA:
+                                        return ASIA_LEVELOPS_CONFIG;
+                                    case SEI_HARNESS:
+                                        return DEFAULT_HARNESS_LEVELOPS_CONFIG;
+                                    case SEI_HARNESS_COMPLIANCE:
+                                        return DEFAULT_HARNESS_COMPLIANCE_LEVELOPS_CONFIG;
+                                    default:
+                                        return DEFAULT_LEVELOPS_CONFIG;
+                                }
+                            }
+
```

Comment:
if you move the values currently in `common` to the enum, this can be replaced with something like:
```
return new LevelOpsConfig(PropeloPluginImpl.getInstance().getApplicationType().getTargetURL())
```

Reply:
Done